api/src/adapters/aws/sesv2_adapter.py

```diff
@@ -22,22 +25,23 @@ class SESV2Response(BaseModel):
 
 class BaseSESV2Client(ABC, metaclass=ABCMeta):
     @abstractmethod
-    def list_suppressed_destinations(self) -> SESV2Response:
+    def list_suppressed_destinations(self, start_date: datetime | None = None) -> SESV2Response:
```

Comment:
If this is a datetime and not a date, could we call this `start_time` instead of `start_date`? I know the underlying call we make calls it that, but that's not well named